tests/integration-tests/tests/common/utils.py

```diff
@@ -121,8 +121,13 @@ def retrieve_pcluster_ami_without_standard_naming(region, os, version, architect
 
 @retry(stop_max_attempt_number=3, wait_fixed=5000)
 def fetch_instance_slots(region, instance_type, multithreading_disabled=False):
-    vcpus = get_instance_info(instance_type, region).get("VCpuInfo").get("DefaultVCpus")
-    return int(vcpus / 2) if multithreading_disabled else vcpus
+    instance_type_info = get_instance_info(instance_type, region)
+    vcpus = instance_type_info.get("VCpuInfo").get("DefaultVCpus")
+    default_threads_per_core = instance_type_info.get("VCpuInfo").get("DefaultThreadsPerCore")
+    if default_threads_per_core is None:
```

Comment:
default should be removed from here, because it's not always true that `default_threads_per_core=2` for `x86_64`. Reason is that there are `x86_64` instance types that have `default_threads_per_core=1`.  
Instead, we should raise an error if `.get("DefaultThreadsPerCore")` cannot be found.